content/millennium/dstu2/medications/medication-administration.md

```diff
@@ -0,0 +1,126 @@
+* [Effective Time](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.effectiveTime_x_){:target="_blank"}
+* [Medication](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.medication_x_){:target="_blank"}
+* Details of medication administered:
+  * [Text](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.text){:target="_blank"}
+  * [Route](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.route){:target="_blank"}
+  * [Quantity](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.quantity){:target="_blank"}
+  * [Site](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.site_x_){:target="_blank"}
+  * [Rate](http://hl7.org/fhir/DSTU2/medicationadministration-definitions.html#MedicationAdministration.dosage.rate_x_){:target="_blank"}
+
+## Search
+
+Search for MedicationAdministrations that meet supplied query parameters:
+
+    GET /MedicationAdministration?:parameters
+
+_Implementation Notes_
+
+* [MedicationAdministration.medication] may be a reference to a [contained] Medication when the Medication cannot be represented by a CodeableConcept because it contains a unique combination of ingredients. Medications in the system always exist within the context of a MedicationAdministration and cannot be referenced independently.
+
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true) %>
+
+
+### Parameters
+
+
+ Name           | Required?          | Type          | Description
+----------------|--------------------|---------------|-----------------------------------------------------------------------------------------------
+`_id`           | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
+`patient`       | This, or `_id`     | [`reference`] | The identifier of the patient who has received the medication administration. Example: `patient=12345`
+`status`        | N                  | [`token`]     | The status of the medication administration, may be a list separated by commas.  Example: `status=active`
+`practitioner`  | N                  | [`reference`] | The identifier of the  performing clinician, may be a list separated by commas. Example: `practitioner=12345`
+`notgiven`      | N                  | [`token`]     | Administrations that were not made. A value of 'true' will search for these, and a value of 'false' will exclude them. Example: `true`
+`effectivetime` | N                  | [`date`]      | The date/time when the Administration was performed. Must use both ge and le prefixes in order to search. Example: `effectivetime=ge2010-08-18&effectivetime=le2015-07-17`
```

Comment:
"Administration" is not a proper noun in this case so "administration" or "Medication Administration" (which is still not a proper noun, but is at least the convention)

Reply:
Went with just plain 'administration': https://github.com/cerner/fhir.cerner.com/pull/169/commits/a18a46531f69088dec1ee6993ef849fb497c772f#diff-39322533700e1bb799f2f6d703a9ae74R57